bin/bats

```diff
@@ -1,3 +1,35 @@
 #! /usr/bin/env bash
```

Comment:
Do we have a preference for `#!/usr` vs `#! /usr`? I don't really care, but just noticed that not all the scripts are consistent.

Reply:
I've tended to prefer `#! /usr`, but don't feel strongly. I think we might need to do one more tiny cleanup pass before releasing to tighten up little details like this.